src/bundle/Resources/public/css/structure.css

```diff
@@ -0,0 +1,36 @@
+html {
+    height: 100%;
+}
+
+body {
+    min-height: 100%;
+    display: flex;
+}
+
+ez-platform-ui-app {
+    display: flex;
+    flex: 1 0 100%;
+    flex-direction: column;
+}
+
+ez-navigation-hub {
+    flex: 0 0;
+}
+
+main {
+    flex: 1 0;
+    display: flex;
+}
+
+#discovery {
+    flex: 0 0 230px;
```

Comment:
We are currently working with these properties in our testing environment, and we'll adjust based on your implementation

```  left: 0;
    display: inline-block;
    width: 115px;
    position: fixed;
    background-color: #565e63;
```